kuma/landing/urls.py

```diff
@@ -24,12 +27,12 @@
 
 urlpatterns = [
     url(r'^contribute\.json$',
-        views.contribute_json,
+        shared_cache_control(views.contribute_json),
```

Comment:
I think it is worth adding a comment on line 27 to explain why you're using the decorators this way as a regular function instead of as an actual decorator in views.py.